ui/app/styles/components/replication-dashboard.scss

```diff
@@ -0,0 +1,71 @@
+.replication-dashboard {
+  .has-margin-bottom {
+    padding-bottom: 9px;
+  }
+
+  .has-large-margin-top {
+    margin-top: 40px;
+  }
+
+  .selectable-card {
+    line-height: normal;
+
+    .title-number {
+      font-size: 28px;
+      align-self: end;
+    }
+  }
+
+  .helper-text {
+    font-weight: normal;
```

Comment:
i just noticed that we can use `$font-weight-normal` here and inside `info-table-row.scss` instead of `normal`. i found the variable in our bulma variables here: https://github.com/hashicorp/vault/blob/master/ui/app/styles/utils/_bulma_variables.scss#L39

Reply:
Thanks!  I'll update.